.github/actions/e2e_sonobuoy/action.yml

```diff
@@ -2,6 +2,9 @@ name: sonobuoy
 description: "Execute the e2e test framework sonobuoy."
 
 inputs:
+  cloudProvider:
+    description: "Necessary for log naming."
```

Comment:
nit: I would add the subject to the description. But maybe that's subjective so feel free to ignore.